### ISSUE TITLE
cannot import name 'DbtOsmosis'

### ISSUE THREAD

**Issue by ludogayet** [innoverio/vscode-dbt-power-user#145]
### Expected behavior

When i click on the "run" button on the VSCode UI, i expect the model to run and print result. As seen in the documentation `Visualize the result set of your model (new)`

### Actual behavior

An error message :

````
cannot import name 'DbtOsmosis' from 'dbt_osmosis.core.osmosis' (/Users/ludo/Library/Caches/pypoetry/virtualenvs/dbt-daIBZcfd-py3.8/lib/python3.8/site-packages/dbt_osmosis/core/osmosis.py)
````

### Steps To Reproduce

Dbt power user : v0.7.9
Dbt-osmosis : 0.8.1 (also tested with 0.8.0)

### Log output/Screenshots

![Capture d’écran 2022-10-03 à 12 09 55](https://user-images.githubusercontent.com/59567967/193552851-d6d379bb-3661-41bd-8693-471c7779c8ab.png)

### Operating System

MacOs Monterey 12.5

### dbt version

1.1.1

### dbt Adapter

dbt-bigquery             1.1.1, dbt-core                 1.1.1, dbt-extractor            0.4.1, dbt-osmosis              0.8.1

### dbt Power User version

0.7.9

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

**Comment by jaypeedevlin** [comment]
I'm also seeing this on Mac with 0.7.9 and the following dbt versions

```
dbt-core==1.2.2
dbt-extractor==0.4.1
dbt-osmosis==0.8.1
dbt-snowflake==1.2.0
```

**Comment by jaypeedevlin** [comment]
@ludogayet I worked this one out, there was a very recent (AKA last few days) refactor and re-release of dbt-osmosis that broke this.  When I downgraded to `dbt-osmosis==0.7.17` things started working for me!

The version is pinned inside the extension's installation command, but if you manage your environments separately (as I do), `pip install dbt-osmosis` will install the version that breaks things.
